src/test/java/BogdanSandu/tests/Tema7/TestDrive.java

```diff
@@ -0,0 +1,4 @@
+package bogdanSandu.tests.Tema7;
```

Comment:
Please remove this class from this PR. You'll add it in PR for homework 7. Thanks!